scripts/openshiftci-presubmit-unittests.sh

```diff
@@ -7,9 +7,16 @@ set -x
 
 export ARTIFACTS_DIR="/tmp/artifacts"
 export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+export PATH=$PATH:$GOPATH/bin
+# set localtion for golangci-lint cache
+# otherwise /.cache is used, and it fails on permission denied
+export GOLANGCI_LINT_CACHE="/tmp/.cache"
 
+make goget-tools
+make validate
+make golint
```

Comment:
According to https://github.com/openshift/odo/pull/2649/files#diff-b67911656ef5d18c4ae36cb6741b7965R61, won't make validate run make golint too?

Reply:
good catch. 
Fixed.